build/container.bzl

```diff
@@ -51,6 +51,7 @@ def multi_arch_container(
         docker_push_tags = None,
         tags = None,
         visibility = None,
+        user = "0",
```

Comment:
Targets the utilise this macro already override `user` to be `1000` (specifically in `build/BUILD.bazel`).

This is the *default* value for this target - previously this was being set as part of `**kwargs` on the *go_image* target *only*. The `container_image` target, which has the timestamp of the build baked in, did *not* set it explicitly and therefore it was being reset back to zero on just those targets.